include/ignition/gui/qml/IgnCard.qml

```diff
@@ -376,11 +485,59 @@ Pane {
         }
         visible: card.showDockButton && !card.standalone
         onClicked: {
-          const docked = card.state === "docked"
+          const docked = (card.state === "docked" || card.state === "docked_collapsed")
           card.state = docked ? "floating" : "docked"
```

Comment:
How do you feel about maintaining the collapsed / expanded state when docking / undocking?

For example, if it's currently `docked_collapsed` and I undock, I'd expect to remain collapsed, so I'd become `floating_collapsed`. The same goes when re-docking.